src/azure-cli/service_name.json

```diff
@@ -0,0 +1,702 @@
+[
+  {
+    "Command": "az account",
+    "Description": "Manage Azure subscription information.",
+    "AzureServiceName": "Azure CLI",
+    "ExtensionName": "",
+    "URL": "https://docs.microsoft.com/azure/cost-management-billing/manage/create-subscription"
+  },
+  {
+    "Command": "az acr",
+    "Description": "Manage private registries with Azure Container Registries.",
+    "AzureServiceName": "Azure Container Registries",
+    "ExtensionName": "",
+    "URL": "https://docs.microsoft.com/azure/container-registry/"
+  },
+  {
```

Comment:
@yungezz , @qwordy , @jiasli , @fengzhou-msft

This is a duplicate comment also found in PR [3083](https://github.com/Azure/azure-cli-extensions/pull/3083)

**Important:** Shuang is already keeping a new JSON file which replaces TitleMapping.json and where this information would fit perfectly. Ideally, we'd like to avoid having engineering enter autogenerated content in multiple places. Can we please check with Alice Wang and Shuang Jiang on a consolidated place for this information?

**Other thoughts:**

1. I feel we should remove "Description" as it is a duplicate of the autogenerated content. There is no reason for making someone enter the same information in two different places, correct?
2. Isn't "extensionName" also a duplicate from what is found in the _help.py or commands.py files? Shuang is currently showing this information at the top of every autogenerated extension page and he doesn't use this new file.
3. URL is not necessary unless we want to a.) take responsibility for maintaining this link or b.) use it in our autogenerated content. I had it in SQL in order to find the Azure service home page for my reference (vs Googling it every time I needed it)
4. I am glad to see "conversational service name" removed -- that was only for my Power BI internal reporting.

Reply:
(1) The information of service_name.json is part of Azure CLI. We seldom put content in Azure CLI Doc repo. (2) TitleMapping.json is aggregated data. service_name.json is raw data. It is easier to read and maintain. Each entry is a command (group).
1. Each command has a description. It is a duplicate info.
2. The extension name is not an explicit value in _help.py or commands.py. It is also used to check which extension does not have an entry.
3. Since we have already written them, I prefer keeping them. Autogenerated content can provide this link to allow readers to jump to the service page.
4. Yes. This column causes confusion.

Reply:
@qwordy, You do realize that TitleMapping.json is going away, correct?  I'm honestly not trying to put my nose into the "how" of the [Azure DevOps work items](https://ceapex.visualstudio.com/Engineering/_workitems/edit/90421) for references, but I do feel that I made a **tone of new work** for engineering that I'd like to minimize so that you all still kind of like me for the next couple of years.

The above link is for the Azure DevOps epic.  Here are the related features which all require new reference metadata content.  Attached to the features is an example JSON file I created in an effort to communicate the request detail:

[Missing extension information](https://ceapex.visualstudio.com/Engineering/_workitems/edit/321675) -- done
[Combine core and extension references on the TOC](https://ceapex.visualstudio.com/Engineering/_workitems/edit/321693) -- the subject of this PR
[Autogen of Azure service reference summaries](https://ceapex.visualstudio.com/Engineering/_workitems/edit/327925)
[Add reference name, command and parameter types, status, version and deprecated message at the parameter level](https://ceapex.visualstudio.com/Engineering/_workitems/edit/327926)
[Catch-all feature for any remaining data-driven metadata not already completed](https://ceapex.visualstudio.com/Engineering/_workitems/edit/327926)

Reply:
@dbradish-microsoft @daxianji007 Are these two files beneficial? My assumption is we maintain service_name.json and .docsreference.azurecli.json is generated. If it is more convenient to keep only one file, it is OK to abandon this PR.

Reply:
@qwordy @dbradish-microsoft I am afraid the file ".docsreference.azurecli.json" cannot be automatically generated. It has some (and will have more) conceptual content, such as TitleMapping and Service Page Description. From my point of view, one file is certainly more convenient for both maintenance and Docs generation. I can also accept spliting the mapping into two service_name.json in 2 repos (actually the new pipeline is ready for both way). It is up to you.

Reply:
@daxianji007, I agree with "the file .docsreference.azurecli.json cannot be autogenerated...", but I am not understanding the "two service_name.json in 2 repos" that we just added.  The engineering team is going to have to maintain 3 different files with this design, correct?  Is there a way to give @yungezz and her team a single data entry file to maintain?

Reply:
@daxianji007 @dbradish-microsoft 
I am OK with both solutions. If one file is more convenient, let's just abandon service_name.json and close these 2 PRs. One last question, who will maintain .docsreference.azurecli.json?

Reply:
@qwordy, I defer to @yungezz or @rloutlaw to answer your question.